set.mm

```diff
@@ -614237,6 +614271,250 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
+       (Contributed by NM, 20-Jan-1997.)  (Revised by Steven Nguyen,
+       7-Jan-2023.) $)
+    rsubadd $p |- ( ( A e. RR /\ B e. RR /\ C e. RR ) ->
+                   ( ( A -R B ) = C <-> ( B + C ) = A ) ) $=
+      ( vx cr wcel w3a crsub co wceq cv caddc crio wb wa rsubval eqeq1d 3adant3
+      wreu rsubeu oveq2 riota2 sylan2 3impb 3com13 bitr4d ) AEFZBEFZCEFZGABHIZC
+      JZBDKZLIZAJZDEMZCJZBCLIZAJZUGUHUKUPNUIUGUHOUJUOCDABPQRUIUHUGURUPNZUIUHUGU
+      SUHUGOUIUNDESUSDBATUNURDECULCJUMUQAULCBLUAQUBUCUDUEUF $.
+  $}
+
+  ${
+    rsubaddd.1 $e |- ( ph -> A e. RR ) $.
+    rsubaddd.2 $e |- ( ph -> B e. RR ) $.
+    rsubaddd.3 $e |- ( ph -> C e. RR ) $.
+    $( Relationship between subtraction and addition.  (Contributed by Mario
+       Carneiro, 27-May-2016.)  (Revised by Steven Nguyen, 8-Jan-2023.) $)
+    rsubaddd $p |- ( ph -> ( ( A -R B ) = C <-> ( B + C ) = A ) ) $=
+      ( cr wcel crsub co wceq caddc wb rsubadd syl3anc ) ABHICHIDHIBCJKDLCDMKBL
+      NEFGBCDOP $.
+  $}
+
+  ${
+    $d x y z $.
+    $( Real subtraction is an operation on the real numbers.  Based on ~ subf .
+       (Contributed by NM, 20-Jan-1997.)  (Revised by Steven Nguyen,
+       7-Jan-2023.) $)
+    rsubf $p |- -R : ( RR X. RR ) --> RR $=
+      ( vy vz vx cv caddc co wceq cr crio wcel wral cxp crsub wf rsubval rsubcl
+      wa eqeltrrd rgen2a df-rsub fmpt2 mpbi ) ADZBDEFCDZGBHIZHJZAHKCHKHHLHMNUFC
+      AHUDHJUCHJQUDUCMFUEHBUDUCOUDUCPRSCAHHUEHMCABTUAUB $.
+  $}
+
+  $( Addition and subtraction of equals.  (Contributed by Steven Nguyen,
+     8-Jan-2023.) $)
+  repncan2 $p |- ( ( A e. RR /\ B e. RR ) -> ( ( A + B ) -R A ) = B ) $=
+    ( cr wcel wa caddc co crsub wceq eqid readdcl simpl simpr rsubaddd mpbiri )
+    ACDZBCDZEZABFGZAHGBISSISJRSABABKPQLPQMNO $.
+
+  $( Addition and subtraction of equals.  Based on ~ pncan3 .  (Contributed by
+     Steven Nguyen, 8-Jan-2023.) $)
+  repncan3 $p |- ( ( A e. RR /\ B e. RR ) -> ( A + ( B -R A ) ) = B ) $=
+    ( cr wcel crsub co caddc wceq rsubcl w3a eqid rsubadd mpbii mpd3an3 ancoms
+    ) BCDZACDZABAEFZGFBHZPQRCDZSBAIPQTJRRHSRKBARLMNO $.
+
+  ${
+    mvrlladdd.1 $e |- ( ph -> A e. RR ) $.
+    mvrlladdd.2 $e |- ( ph -> B e. RR ) $.
+    mvrlladdd.3 $e |- ( ph -> ( A + B ) = C ) $.
+    $( Move real LHS left addition into RHS (real subtraction).  (Contributed
+       by Steven Nguyen, 8-Jan-2023.) $)
+    mvrlladdd $p |- ( ph -> B = ( C -R A ) ) $=
+      ( crsub co cr wcel wceq caddc readdcld eqeltrrd rsubadd syl5ibrcom mp3and
+      w3a eqcomd ) ADBHIZCADJKZBJKZCJKZUACLZABCMIZDJGABCEFNOEFAUEUBUCUDSUFDLGDB
+      CPQRT $.
+  $}
+
+  ${
+    mvrlladdi.1 $e |- A e. RR $.
+    mvrlladdi.2 $e |- B e. RR $.
+    mvrlladdi.3 $e |- ( A + B ) = C $.
+    $( Move real LHS left addition into RHS (real subtraction).  (Contributed
+       by Steven Nguyen, 8-Jan-2023.) $)
+    mvrlladdi $p |- B = ( C -R A ) $=
+      ( crsub co wceq wtru cr wcel a1i caddc mvrlladdd mptru ) BCAGHIJABCAKLJDM
+      BKLJEMABNHCIJFMOP $.
+  $}
```

Comment:
Unless you have something specific in mind, I would say it is probably useless to have the inference ~mvrlladdi when you already have the deduction ~mvrlladdd.